void tomo::IoLoop::run_loop<true, true, false, false, false>()
--- PRE

+++ POST

@@ -636,7 +636,7 @@

 0bcd [ 3] nopl (%rax)
 0bd0 [ 5] mov $0x1,%esi
 0bd5 [ 3] mov %rbx,%rdi
-0bd8 [ 5] call <tomo::IoLoop::arm_accept(tomo::UrKind)>
+0bd8 [ 5] call <tomo::IoLoop::fliplisten_retry_accept(tomo::UrKind)>
 0bdd [ 5] jmp <+0x2d2>
 0be2 [11] data16 cs nopw 0x0(%rax,%rax,1)
 0bed [ 3] nopl (%rax)
